### ISSUE TITLE
Use of NVDA screen reader disables keyboard navigation in mat-tree

### ISSUE THREAD

**Issue by revittek** [angular/components#13635]
#### Bug, feature request, or proposal:
When NVDA screen reader is on, navigation using tab, shift-tab, space, enter, and up and down arrows does not work as expected.

#### What is the expected behavior?
While using a screen reader such as NVDA, to be able to navigate through a tree, select or open and close drawers using the keyboard.

#### What is the current behavior?
Enter key specifically does not work - it is impossible to select a checkbox in the checkbox tree example.

#### What are the steps to reproduce?
Video: https://www.youtube.com/watch?v=0cQHC8d4y10

To reproduce:
1) Install NVDA - https://www.nvaccess.org/download/
2) Turn on NVDA
3) Visit Angular Material Tree examples - https://material.angular.io/components/tree/examples
4) Tab to the caret in "Tree with checkboxes" example
5) When focus is on the caret, press spacebar - drawer should open
6) Tab to almond meal flour checkbox, and press spacebar - checkbox should be checked

#### What is the use-case or motivation for changing an existing behavior?
Angular Material should take steps to ensure that functionality works for users with screen readers in order to be accessible.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
6.4.7

**Comment by crisbeto** [comment]
AFAIK the tree doesn't have keyboard support at the moment. https://github.com/angular/material2/pull/10691 was supposed to add it. cc @jelbourn

**Comment by jelbourn** [comment]
Yeah, this is something on the backlog for both tree and table. Neither of them include keyboard shortcuts automatically, but we wanted to include add-on directives that would bring in the keyboard interaction.

**Comment by divyanshumehta** [comment]
Hey @jelbourn  is this feature in the making any time near, because I really need keyboard navigation or should I look toward taking a different approach which does not involve tree.

**Comment by JaxonWright** [comment]
In my experience, this problem only occurs with trees that have a flattened structure. With those trees, the first group of nodes is the only one that can be expanded and selected. 

The ones with nested nodes work as I would expect them.

However, I do not use NVDA for anything other than testing and am not an actual daily user of it, so what I expect might not be how it is supposed to work.

**Comment by mfairchild365** [comment]
This doesn't appear to be a feature request. Requiring voting on accessibility defects from the community is not appropriate and is discriminatory. Since that comment was made by a bot, I don't know if it was intentional. Either way, please reconsider your approach. CC @jelbourn @crisbeto 

That being said, it appears that this specific bug may already be resolved.
